api/api_models/user_response.py

```diff
@@ -7,7 +8,22 @@ class UserResponse(BaseModel):
     email: EmailStr = Field(...)
     first_name: str = Field(...)
     last_name: str = Field(...)
+    github_profile: str=Field('')
+    twitter_profile: str=Field('')
+    linkedin_profile: str=Field('')
+    portfolio_url: Optional[str]=Field('')
+    profile_pic_url: str=Field('')
     created_at: datetime = Field(...)
 
     class Config:
         orm_mode = True
+
+class ProfileResponse(BaseModel):
+    email: Optional[EmailStr] 
+    first_name: Optional[str] 
+    last_name: Optional[str] 
+    github_profile: Optional[str] 
+    twitter_profile: Optional[str] 
+    linkedin_profile: Optional[str] 
+    portfolio_url: Optional[str]
```

Comment:
Clean up model name and how it's used.
Also consider refactoring existing model that could be reused here.